cmd/snap-confine/ns-support.c

```diff
@@ -700,6 +707,14 @@ static void helper_capture_per_user_ns(struct sc_mount_ns *group, pid_t parent)
 	debug("capturing per-snap, per-user mount namespace");
 	sc_must_snprintf(src, sizeof src, "/proc/%d/ns/mnt", (int)parent);
 	sc_must_snprintf(dst, sizeof dst, "%s.%d.mnt", group->name, (int)uid);
+
+	/* Ensure the bind mount destination exists. */
+	int fd = open(dst, O_CREAT | O_CLOEXEC | O_NOFOLLOW | O_RDONLY, 0600);
```

Comment:
The old code was using openat() the new one is using plain open() - does this change anything? should openat(AT_FDCWD) be used here?

Reply:
open is equivalent to openat(AT_FDCWD), I don't know if we would prefer one over the other.

Reply:
Note that the original code ran in a different moment, it used openat because it held a descriptor open to the base directory. This is running from a helper process that we fork that moves to the proper directory and stays there.

Reply:
Thats fine, I really just asking because I noticed it :)